src/App.tsx

```diff
@@ -2,97 +2,14 @@
 import React from 'react';
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
 
 export const App: React.FC = () => (
   <div className="page">
     <div className="page-content">
       <div className="movies">
```

Comment:
this div should also be extracted to the movies list, they should display with grid, now its a litle broken
<img width="1332" alt="image" src="https://user-images.githubusercontent.com/66749919/191317297-780d8d94-8be8-49b2-bcf4-7daa0f4144c5.png">

Reply:
chenged